ocs_ci/templates/workloads/hsbench/hsbench_obj.yaml

```diff
@@ -10,6 +10,6 @@ num_bucket: 1
 num_obj: 1000000
 run_mode: "cxip"
 bucket_prefix: "bp01"
-end_point: "http://rook-ceph-rgw-ocs-storagecluster-cephobjectstore.openshift-storage.svc.cluster.local"
+end_point: "http://rook-ceph-rgw-ocs-storagecluster-cephobjectstore.openshift-storage.svc.cluster.local:80"
 end_point_port: 80
```

Comment:
if end_point is updated with the port, do we need `end_point_port` to be defined?

Reply:
To simplify the endpoint string for rgw, I combine end_point URL and and endpoint_port in once.  
the end_point URL alone will be used to stream I/O to noobaa bucket. 

end_point_port can be removed.